dev/custom-pages/space.scss

```diff
@@ -0,0 +1,10 @@
+.ucf-section-space-ignite-callout .ignite-logo {
```

Comment:
We may want to consider adding the .ignite-logo class to the theme's core css, since it's used on at least one other page section (the home page)...it'd be easier to have it in one place than having to copy the same styles onto multiple pages or sections.